agt/state.py

```diff
@@ -44,7 +53,7 @@ def __init__(self, output_callback):
 
         self.memory = {}
 
-    async def say(self, text: str, image_url: str = None):
+    async def say(self, message: Message):
```

Comment:
this is not good it breaks all the existing components

Reply:
Should it be backwards compatible, leaving the text and the image as a parameters and add additional pydantic scheme for ssml and future add-ons? I planned to adjust all the components and started this change as a draft.